src/experiments.js

```diff
@@ -121,3 +138,13 @@ function saveExperimentIds(win, experimentIds) {
   setCookie(win, COOKIE_NAME, experimentIds.join(','),
       timer.now() + COOKIE_EXPIRATION_INTERVAL);
 }
+
+/**
+ * Resets the experimentsToggle cache for testing purposes.
+ * @visibleForTesting
+ */
+export function resetExperimentToggles_() {
```

Comment:
We don't have any way to somehow run it in afterEach for all tests automatically?